src/components/seo.js

```diff
@@ -24,43 +24,43 @@ function SEO({ description, lang, meta, keywords, title }) {
       }}
       meta={[
         {
-          name: `description`,
-          content: 'A place for Q&A and free mentorship',
+          name: `description',
```

Comment:
Missed one of those back ticks 😃
```suggestion
          name: 'description',
```